src/components/MapFieldValues/MapFieldValues.tsx

```diff
@@ -117,6 +117,19 @@ export const MapFieldValues = ({
               content = <H2>{usedField.map((e: any) => e.name).join(",")}</H2>;
             } else if (field.schema.items === "option") {
               content = <H2>{usedField.map((e: any) => e.value).join(",")}</H2>;
+            } else if (field.schema.items === "issuelinks") {
+              const value = issue[field.key] as any[];
```

Comment:
We should use `any`, this looks like it should
```
{
    inwardIssue?: {
        key?: string
    }
}[]
```

Reply:
fixed